### PR TITLE
Adding chlab-10-3 MNIST and MLP with Dropout

### PR DESCRIPTION
**Pull request by surfertas**
Added chlab-10-3 deeper MLP with dropout trained on MNIST.

### CODE REVIEW

**Comment by kkweon** [comment]
Do we need `argparse`? 
As far as I know, 
- we didn't define any argparse in Tensorflow
- save any model except in [lab-13-3-mnist_save_restore.py](https://github.com/hunkim/DeepLearningZeroToAll/blob/master/lab-13-3-mnist_save_restore.py)

In general, you'd need a very good reason for breaking consistency.
I guess it's okay since other chainer files already have these.
Ignore this if it's been already discussed.

Otherwise, it looks good to me 👍  except 
```python
parser.add_argument('--resume',
                    '-r',
                    default='',
                    help='Resume the training from snapshot')
```
I wish the help description was clear that a path/to/snapshot should be given as an input.
Currently, it feels like it's a boolean flag. 

cc @hunkim

**Comment by hunkim** [comment]
If we can avoid it, it would be the best.

**Comment by surfertas** [comment]
Hi yes, we can avoid if that is the preferred template. I was just trying to be relatively consistent with the PFN provided examples here.

https://github.com/pfnet/chainer/tree/master/examples

If you prefer to be consistent with other labs I am happy to remove and replace with hard coded values.

No issues either way. Please advise.
